Modules/overlapped.c

```diff
@@ -1451,8 +1728,10 @@ static PyMethodDef overlapped_functions[] = {
     {"ResetEvent", overlapped_ResetEvent,
      METH_VARARGS, ResetEvent_doc},
     {"ConnectPipe",
-     (PyCFunction) ConnectPipe,
+     (PyCFunction)ConnectPipe,
```

Comment:
Looks like accidental space removing.
Moreover, the cast to `PyCFunction` is not needed: we don't do it for other functions.

The same for `overlapped_WSAConnect`